### DISCUSSION TITLE
〔 Comments Section 〕Fallout Shelter Online: Trinkets

### DISCUSSION THREAD

**Discussion by morgosus** [morgosus/morgosus.github.io#13] [Comments]
A comments section for the [Fallout Shelter Online: Trinkets](https://www.toms.click/gaming/foso/2021/08/13/fallout-shelter-trinkets.html) post.

**Comment by Emptycz** [comment]
Are there any dwellers with a trinket who aren't in the VR room?

**Reply by morgosus** [reply]
Yes. The only dweller that comes to mind that's like that is Khan Maykr who comes from the Doom crossover events. The other two are in the VR room (Doom Slayer and Marauder).

One way to obtain her trinkets would be through drawing in the wish pool, or wilds wanderer pool, when they're available.

**Comment by morgosus** [comment]
Desdemona medal 5 guide: Keep bomb, use auto to kill squad, switch to manual, kill top left using auto, switch to manual, grab the helper on the left, use extinguisher on the guys below use auto (or a nuke - you won't need it elsewhere) to kill them now, your support will die, desdemona will grab the key and get another helper though, use the key and unlock the gategrab the hi-capacity fusion cell (in next combat energy +50%), use stealth boy to get past the next enemy, grab the 50% boost again, and bypass the next boss. Find the stimpak and heal desdemona, pick up AP boost, grab another +50% damage boost, bypass the next strong enemy and attack the boss. You should have 100% AP on desdemona, you can use med-x to boost one of the 2 support guys you have with you. Make sure to switch to x1 speed before entering the boss - there is a dialogue that can make you attack the expert enemy instead.

**Reply by morgosus** [reply]
Spam click the stealth boy as you enter the combat to use it asap. Also, you can use x1 speed if you're not confident you can do it quickly enough.
